tests/integ/modin/test_concat.py

```diff
@@ -25,56 +25,56 @@
 
 @pytest.fixture(scope="function")
 def df1():
-    return pd.DataFrame(
+    return native_pd.DataFrame(
```

Comment:
I changed this because with a Snowpark pandas DataFrame being returned the query count is higher and `to_pandas()` is called a lot. I think it's better to just convert the object to Snowpark pandas whenever needed